endpoints/api/namespacequota.py

```diff
@@ -143,12 +160,36 @@ class OrganizationQuota(ApiResource):
         "UpdateOrgQuota": {
             "type": "object",
             "description": "Description of a new organization quota",
-            "properties": {
-                "limit_bytes": {
-                    "type": "integer",
-                    "description": "Number of bytes the organization is allowed",
+            "oneOf": [
+                {
+                    "properties": {
+                        "limit_bytes": {
+                            "type": "integer",
+                            "description": "Number of bytes the organization is allowed",
+                        },
+                    },
+                    "required": ["limit_bytes"],
+                    "additionalProperties": False,
                 },
-            },
+                {
+                    "properties": {
+                        "limit": {
+                            "type": "string",
+                            "description": "Human readable storage capacity of the organization",
+                            "pattern": r"^(\d+\s?(B|KiB|MiB|GiB|TiB|PiB|EiB|ZiB|YiB|Ki|Mi|Gi|Ti|Pi|Ei|Zi|Yi|KB|MB|GB|TB|PB|EB|ZB|YB|K|M|G|T|P|E|Z|Y)?)$",
+                        },
+                    },
+                    "required": ["limit"],
+                    "additionalProperties": False,
+                },
+                {
```

Comment:
what does this do?

Reply:
Currently, the API, for whatever reason, allows a valid request to be submitted that does not contain `limit_bytes`. This schema makes it so that either `limit` or `limit_bytes` or neither of them can be supplied, essentially allowing an empty request. I don't necessarily understand why the parameter `limit` can be omitted since the API call doesn't seem to do anything useful in this case, but I needed to retain this behavior since this API is released fora while and clients may rely on it.